Code/GraphMol/FileParsers/SDMolSupplier.cpp

```diff
@@ -267,6 +267,7 @@ unsigned int SDMolSupplier::length() {
     // now remember to set the stream to the last position we want to read
     dp_inStream->clear();
     dp_inStream->seekg(d_molpos[d_last]);
+    df_end = false;
```

Comment:
This doesn’t belong in this pr

Reply:
@bp-kelley Yeah, not sure what I messed up in the previous push. I have re-pushed and now it's OK.